app/src/main/res/drawable/start_button_primary_background.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+  <item>
+    <layer-list>
+      <item android:right="2dp" android:top="2dp">
+        <shape>
+          <corners android:radius="4dp" />
+          <solid android:color="#D6D6D6" />
```

Comment:
`Optional Nit: ` Shift this color to colors.xml because I think this color will be common at other places too.

Reply:
Check `grey_shade_20` in colors.xml, you can use that color directly here.

Reply:
done.